crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -846,6 +850,24 @@ type TransactionSignature {
 	base64Sig: Base64!
 }
 
+"""
+Information about which previous versions of a package introduced its types.
+"""
+type TypeOrigin {
+	"""
+	Module defining the type.
+	"""
+	module: String!
+	"""
+	Name of the struct.
+	"""
+	struct: String!
+	"""
+	The storage ID of the package that first defined this type.
+	"""
+	definingId: SuiAddress!
```

Comment:
Thought: the linkage field has original(runtime) and upgradedId (storage) - would it be confusing to rename upgradedId and definingId to storageId?

Reply:
It potentially makes sense to rename `linkage`'s fields to `runtimeId` and `storageId` -- that's how they are named in VM internals, but I chose this naming because in `sui-types` that's the names these fields got.

But it doesn't make sense to make `definingId` also `storageId`, because for a given type, it's worth distinguishing the ID that it gets referred to at runtime (`originalId` or `runtimeId`), the ID of the package it was introduced by (`definingId`), and the ID of the package it is currently being used in (`upgradedId` or `storageId`).